lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2354,6 +2354,8 @@ llvm::Triple SwiftASTContext::GetSwiftFriendlyTriple(llvm::Triple triple) {
     if (triple.isOSLinux() &&
         triple.getEnvironment() == llvm::Triple::UnknownEnvironment)
       triple.setEnvironment(llvm::Triple::GNU);
+    if (triple.isOSWindows() && triple.isWindowsMSVCEnvironment())
```

Comment:
Could you add a comment here explaining why this is being done/necessary?

Reply:
Sure, though I would say that we should have a comment in the case of the apple targets underneath as well then:

https://github.com/apple/llvm-project/pull/1794/files/77cc292b94e17332e894ba3f83799c91cbddde82#diff-f34d8da7134dc1075142dd71385a3045R2363-R2369

The entire purpose of this function is to convert the triple to match the Triple for the Swift runtime.